openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -100,17 +114,20 @@ function renderConferenceTabs() {
     heading: 'Your Consoles',
     id: 'your-consoles',
   });
-  var tabNames = new Set();
-  for (var decision in DECISION_HEADING_MAP) {
-    tabNames.add(DECISION_HEADING_MAP[decision]);
-  }
-  var tabArray = Array.from(tabNames);
-  tabArray.forEach(function(tabName) {
+
+  var tabNames = []
+
+  Object.keys(DECISION_HEADING_MAP).forEach(function(key) {
+    venueIds.push(key)
+    tabNames.push(DECISION_HEADING_MAP[key])
+  })
+
+  tabNames.forEach(function(tabName) {
```

Comment:
It would be good to combine this forEach with the one above. The tabNames array isn't needed outside of that forEach, so you could remove the array and just use a local var inside the function.